pkg/services/accesscontrol/models.go

```diff
@@ -96,13 +96,13 @@ const (
 	ActionPluginsManage = "plugins:manage"
 
 	// Global Scopes
-	ScopeGlobalUsersAll = "global:users:*"
+	ScopeGlobalUsersAll = "global:users:all"
 
 	// Users scope
-	ScopeUsersAll = "users:*"
+	ScopeUsersAll = "users:all"
 
 	// Settings scope
-	ScopeSettingsAll = "settings:**"
+	ScopeSettingsAll = "settings:all"
```

Comment:
These are used in the declaration of predefined roles as well, so access to individual resources would break with this change (though you might not notice that when testing locally, because the version of the roles isn't incremented, so it's not modified).

Reply:
I have started to break the roles into two types, "resource roles", and "user roles".  Resource roles can't have wildcards and user roles can.  The current testing framework doesn't really support this, I need to modify a lot of things